src/extension-runner-local/src/bin/main.rs

```diff
@@ -14,8 +14,11 @@ fn main() {
 #[derive(Debug, StructOpt)]
 pub enum RunnerCmd {
     /// Run a Streaming Controller (SC) or SPU
-    #[structopt(name = "run")]
+    #[structopt()]
     Run(RunCmd),
+    /// Run a new Streaming Processing Unit (SPU)
+    #[structopt()]
```

Comment:
Is this supposed to have a short name?

Reply:
No

Reply:
Gotcha. Ok, then this attribute isn't really necessary :P but it also doesn't hurt really